### ISSUE TITLE
Dependency conflict: rive and share_plus

### ISSUE THREAD

**Issue by eli1stark** [rive-app/rive-flutter#415]
### Problem Description

There is currently a dependency conflict between `rive` and `share_plus` packages due to incompatible versions of the `web` package.

The error message when trying to resolve dependencies is as follows:

```
Resolving dependencies...
Because rive 0.13.12 depends on rive_common 0.4.10 which depends on web ^0.5.1, rive 0.13.12 requires web ^0.5.1.
And because share_plus >=10.0.1 depends on web ^1.0.0, rive 0.13.12 is incompatible with share_plus >=10.0.1.
So, because offsuit depends on both share_plus 10.0.2 and rive 0.13.12, version solving failed.
```

This conflict prevents developers from using both `rive` and `share_plus` in the same project, which limits the functionality available to Flutter applications.

### Current Versions

- rive: 0.13.12
- share_plus: 10.0.2
- web (required by rive): ^0.5.1
- web (required by share_plus): ^1.0.0

### Possible Solutions

1. **Update rive_common dependency**: 
   Update the `rive_common` package to use `web ^1.0.0` instead of `web ^0.5.1`. This would allow `rive` to be compatible with the newer version of `web` used by `share_plus`.

2. **Create a new release of rive**:
   Release a new version of `rive` that depends on an updated version of `rive_common` which is compatible with `web ^1.0.0`.

3. **Loosen web package constraints**:
   If possible, loosen the constraints on the `web` package in `rive_common` to allow for a wider range of versions, including both `^0.5.1` and `^1.0.0`.

### Impact

This issue affects developers who want to use both Rive animations and the sharing functionality provided by `share_plus` in their Flutter applications. Resolving this conflict would greatly improve the developer experience and allow for more feature-rich applications.

### Additional Information

- Flutter version: [3.24.1 ]
- Dart version: [3.5.1]

Thank you for your attention to this matter. We appreciate your work on the Rive package and look forward to a resolution that allows for better compatibility with other popular Flutter packages.

**Comment by eli1stark** [comment]
@HayesGordon https://github.com/rive-app/rive-flutter/issues/413

**Comment by HayesGordon** [comment]
This should now be resolved in v0.13.13

We've increased the dependency range.

Please reopen if you're still experiencing issues after upgrading.
